VKApp/VKApp/Views/SearchGroupTableViewCell.swift

```diff
@@ -23,6 +23,7 @@ final class SearchGroupTableViewCell: UITableViewCell {
     // MARK: - Public Methods
 
     func configureCell(group: Group) {
+        selectionStyle = .none
         groupNameLabel.text = group.groupName
         groupPhotoImageView.image = UIImage(named: group.groupPhotoName)
```

Comment:
```suggestion
        groupPhotoImageView.image = getImage(by name: String)       
```
так выглядеть будет красивее, если везде вызывать метод
func getImage(by name: String) -> UIImage {
     UIImage(named: name)
}